doc/man3/PKCS12_gen_mac.pod

```diff
@@ -31,10 +35,12 @@ PKCS12_setup_mac() sets the MAC part of the PKCS#12 structure with the supplied
 parameters.
 
 PKCS12_set_mac() sets the MAC and MAC parameters into the PKCS#12 object.
+PKCS12_set_pbmac1_with_pbkdf2() sets the MAC and MAC parameters into the
+PKCS#12 object when B<PBMAC1> is used for protection of the PKCS#12 object.
 
 I<pass> is the passphrase to use in the HMAC. I<salt> is the salt value to use,
 I<iter> is the iteration count and I<md_type> is the message digest
-function to use.
+function to use. I<prf_md_name> specifies the digest used for PBMAC1 KDF.
```

Comment:
...the digest used for the PBKDF2 in PBMAC1

Reply:
Fixed